azure-pipelines.yml

```diff
@@ -59,8 +59,8 @@ jobs:
   - script: |
       set -e
       export PKG_CONFIG_PATH=/usr/local/opt/libffi/lib/pkgconfig
-      opam init -a -j "$NJOBS" --compiler=$COMPILER
-      opam switch set $COMPILER
+      opam init -a -j "$NJOBS" --compiler=ocaml-base-compiler.$COMPILER
+      opam switch set ocaml-base-compiler.$COMPILER
```

Comment:
If you can tweak this to be `ocaml.$COMPILER` would be great, but the merge shouldn't be blocked on that.